Project1/Task8-homework/Task8.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Project1.Task7_homework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project1.Task8_homework
+{
+    public class Task8
+    {
+        public static void Runner()
+        {
+            try
+            {
+                ShowMassiveElement();
+            }
+            catch (IndexOutOfRangeException ex)
+            {
+                Console.WriteLine(ex.Message);
```

Comment:
look, you have duplicating strings bellow, create method with Exception parameter and put you exception there, call this new method in catch block instead of duplicating